portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,122 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let eerieModeOn = false; 
+
 /**
- * Adds a random greeting to the page.
+ * Shows and hides "Eerie Mode" elements
```

Comment:
Not sure how to do this in Github but I'm curious what the eerie mode looks like.